responses.py

```diff
@@ -622,6 +622,7 @@ def _find_match(self, request):
     def _on_request(self, adapter, request, **kwargs):
         match = self._find_match(request)
         resp_callback = self.response_callback
+        request.params = dict(parse_qsl(request.path_url.replace("/?", "")))
```

Comment:
Could we include this in the example under Dynamic Responses?

Reply:
Example has been added.